scripts/chocolatey/coder.nuspec

```diff
@@ -22,13 +22,14 @@
     <title>Coder (Install)</title>
```

Comment:
```suggestion
    <title>Coder</title>
```
Why do we have (Install) as suffix here?

Reply:
It's a pretty standard thing, but we don't need it, I did it on accident

Reply:
removed in f7d2695